src/DiscordBot.js

```diff
@@ -178,7 +178,7 @@ class DiscordBot {
                     break;
                 case "!roverbindgrouprank":
                     if (argument.length > 0) {
-                        let bindArgs = argument.split(':');
+                        let bindArgs = argument.split(' ');
```

Comment:
I purposefully chose `:` as a delimiter because many role names use spaces. See #17 for my solution i'm working on